locksmith/src/controllers/v2/verifierController.ts

```diff
@@ -0,0 +1,185 @@
+import type { Web3Service } from '@unlock-protocol/unlock-js'
+import { Request, Response } from 'express'
+import Normalizer from '../../utils/normalizer'
+import logger from '../../logger'
+import { Verifier } from '../../models/verifier'
+
+export class VerifierController {
+  public web3Service: Web3Service
+
+  constructor({ web3Service }: { web3Service: Web3Service }) {
+    this.web3Service = web3Service
+  }
+
+  private async checkIsLockManager(
```

Comment:
Don't use `private` keyword for methods in typescript. If you want to make a method private, use `#` while defining the name. The private keyword in typescript doesn't offer any guarantee of the method not being accessible publicly.

Reply:
Also I think we may already have a function for this in the code base? Probably better to isolate and move to a helper file?

Reply:
We do have a similar function for the metadata lock endpoint because only the lock manager can operate on that. A middleware abstracting the logic would be good, I think.

Reply:
That is a great idea @kalidiagne ^^